recipes/quickstart/README.md

```diff
@@ -2,6 +2,7 @@
 
 If you are new to developing with Meta Llama models, this is where you should start. This folder contains introductory-level notebooks across different techniques relating to Meta Llama.
 
+* The [Build_with_Llama 3.2](./build_with_Llama_3_2.ipynb) notebook showcases a comprehensive walkthrough of the new capabilities of LLaMA 3.2 models, including multimodal use cases, function/tool calling, LLaMA Stack, and LLaMA on edge.
```

Comment:
We should always use standard naming `Llama`

Reply:
sorry, fixed